targets/UnrealMarketplacePlugin/examplesource/ExampleProject/Plugins/PlayFab/Source/PlayFabCommon/Public/TestFramework/PlayFabTestContext.h.ejs

```diff
@@ -115,7 +115,7 @@ public:
     UFUNCTION()
     void EndTest(PlayFabApiTestFinishState InFinishState = PlayFabApiTestFinishState::PASSED, FString InMessage = "")
     {
-        UE_LOG(LogPlayFabCommon, Log, TEXT("Ending Test: %s."), *testName);
+        UE_LOG(LogTemp, Log, TEXT("Ending Test: %s."), *testName);
```

Comment:
I think that Unreal will possibly reject us if we don't use a properly defined log.
Maybe they won't catch us, but the point being, our customers must be able to identify that a log comes from us.

Reply:
Note that we already use LogTemp in this file up top.

Since it is our test code, it should never show up in a customer's game runtime environment, so it shouldn't interfere with whatever output they are logging. Unreal shouldn't reject us on this basis, have they mentioned this previously when we were submitting PlayFab to the store?

Reply:
OIC, if it's in the tests and not part of the Mkpl plugin, then that's probably fine.